libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -135,6 +135,9 @@ class AC_WPNav
     /// reached_destination - true when we have come within RADIUS cm of the waypoint
     virtual bool reached_wp_destination() const { return _flags.reached_destination; }
 
+    /// reached_destination - true when we have come within RADIUS cm of the waypoint
```

Comment:
looks like the comment should be updated (currently a copy-paste of the function above).